src/toolkit/matl_buy_policy.cc

```diff
@@ -43,18 +45,44 @@ void MatlBuyPolicy::set_manager(Agent* m) {
   }
 }
 
+void MatlBuyPolicy::set_total_inv_tracker(TotalInvTracker* t) {
+  if (t == NULL){
+    std::vector<ResBuf<Material>*> bufs = {buf_};
+    buf_tracker_->Init(bufs, buf_->capacity());
+  }
+  else {
+    buf_tracker_ = t;
+  }
+}
+
+void MatlBuyPolicy::set_inv_policy(std::string inv_policy, double fill, double req_at) {
+  set_req_at(req_at);
+  if ((inv_policy == "sS") || (inv_policy == "Ss")) {
+    set_fill_to(fill);
+  }
+  else if ((inv_policy == "RQ") || (inv_policy == "QR")) {
```

Comment:
Do you want to handle lower case?
```suggestion
  std::transform(inv_policy.begin(), inv_policy.end(), inv_policy.begin(), ::tolower);
  if ((inv_policy == "ss")) {
    set_fill_to(fill);
  }
  else if ((inv_policy == "rq") || (inv_policy == "qr")) {
```